mixer/pkg/attribute/envoyProtoBag.go

```diff
@@ -92,7 +92,7 @@ func stripPrincipal(principal string) string {
 func AuthzProtoBag(req *authz.CheckRequest) *EnvoyProtoBag {
 	pb := envoyProtoBags.Get().(*EnvoyProtoBag)
 	reqMap := make(map[string]interface{})
-	reqMap["context.reporter.kind"] = "inbound"
+	reqMap["context.reporter.kind"] = "unknown"
```

Comment:
The `ExtAuthz::check` should be `inbound` no matter what, right?  what is gained by setting it to `unknown`?

Reply:
It is actually `outbound` at ingress gateway, so I updated it to `unknown`. I don't expect that anyone would reference this attribute in their check instance, so I am fine to use either `unknown` or `inbound`.

Reply:
gateway behavior bites again. the issue will be if anyone is using APAs or similar that are selected based on inbound/outbound.  but `unknown` seems fine to start with, i guess.